packages/ember-glimmer/lib/component-managers/curly.ts

```diff
@@ -73,27 +73,26 @@ function applyAttributeBindings(
   component: Component,
   operations: ElementOperations
 ) {
-  let seen: string[] = [];
+  let seen: Set<String> = new Set();
```

Comment:
I doubt Set to be actually faster for less than < 10 attributes and most elements don't have that many.

We should definitely try to test this.

Reply:
such thought crossed my mind too, so reverting this changes :P